### DISCUSSION TITLE
how can I change the bearing of a Marker's icon in MapLibre Native for android 10.0.2

### DISCUSSION THREAD

**Discussion by cc-shifo** [maplibre/maplibre-native#2578] [Q&A]
Requirements
A marker on map which represents a drone, and the marker's postion will be changed in real time according to the drone's current location(lattidue, longitude, height) and attitude(heading).

Questions
After I call MapboxMap#addMarker(MarkerOptions) to add a marker on map, how can I update the rotation or bearing of this icon?

**Comment by wipfli** [comment]
I am not sure how to do this with markers, but with symbol icons you can have a geojson source for the icon and then just update the source. This is how I update locations of hot-air balloons on a map...
